node/hack/governor/src/index.ts

```diff
@@ -290,5 +300,71 @@ axios
     );
   })
   .catch((error) => {
-    console.error(error);
+    console.error("Request error:", error);
   });
+
+
+/*
+  Perform type checks on the incoming values
+  Check for a denylist set of characters
+
+  If either of these fail, we reject adding the token.
+
+  Example data: 30 000000000000000000000000b5c457ddb4ce3312a6c5a2b056a1652bd542a208 O404 omni404 18 1128.69 0xb5c457ddb4ce3312a6c5a2b056a1652bd542a208 7.4832146999999996
+*/
+async function safetyCheck(chain, wormholeAddr, symbol, coinGeckoId, tokenDecimals, tokenPrice, address, notional)  : Promise<boolean>{
```

Comment:
Dumb question but what's the reasoning for making these functions asynchronous?

Reply:
Good question!

Honestly, I've had so many issues over the years trying to write JavaScript with promises that I just have async and awaits on most things to avoid dealing with async issues. It's super hard to debug and I hate it.

I just ran a test without all of the async stuff and it works fine. I can remove it to simplify things.